--- org/include/vlc_fixups.h	2016-09-26 04:06:38.000000000 +0200
+++ mod/include/vlc_fixups.h	2016-10-02 16:46:09.548650867 +0200
@@ -33,7 +33,7 @@
 
 /* C++11 says there's no need to define __STDC_*_MACROS when including
  * inttypes.h and stdint.h. */
-#if defined (__cplusplus) && (defined(__MINGW32__) || defined(__UCLIBC__))
+#if defined (__cplusplus)
 # ifndef __STDC_FORMAT_MACROS
 #  define __STDC_FORMAT_MACROS 1
 # endif
